application/users/user_schema.py

```diff
@@ -1,14 +1,16 @@
 """Defines schema for database Team objects"""
 from marshmallow import fields, post_dump
-
 from application.base_schema import BaseSchema, output_decorator
+from application.users.oauth.google_oauth_schema import GoogleOAuthSchema
 
 
 class UserSchema(BaseSchema):
     """Schema for importing and exporting Team objects"""
     id_ = fields.UUID(missing=None)
     username = fields.String()
     email = fields.String()
+    google = fields.Nested(GoogleOAuthSchema)
+    has_password = fields.Bool()
```

Comment:
do we need to export has_password?

Reply:
yeah :( In the settings page a user can change the password, if he doesn't have a password because it's a google user, that shouldn't be rendered